readme.md

```diff
@@ -45,11 +43,18 @@ Returns the result `Object`.
 ## Result
 
 - `title` - Window title
-- `id` - Window ID
+- `id` - Window ID (in MacOS and Linux)
 - `app` - App owning the window
 - `pid` - Process ID of the app owning the window
 
 
+## Operating system support
+
+It works with MacOS, Linux and Windows (tested in Windows 7 and above).
```

Comment:
MacOS => macOS

---

> Windows (tested in Windows 7 and above)

Just say `Windows 7+`. I'm not interested in supporting anything lower anyways.